docs/source/conf.py

```diff
@@ -68,7 +68,7 @@
                             "undoc-members": True,
                             "private-members": True,
                             # "special-members": True,
-                            "inherited-members": True,
+                            "inherited-members": False,
```

Comment:
What made you change this? Also, if this was true before this, the parsers documentation must have been a mess...

Reply:
Exactly. I was looking at the built documentation for SmartLoc and it was massive because it included every single `NavData` function since `SmartLoc` is a child of `NavData`. I thought that was unnecessary, so I changed this parameter so it only shows the `SmartLoc` specific functions. The documentation still says how it's a child of `NavData`, so I figure people can look directly at the `NavData` page if they're interested.